tests/src/linearElasticityTest.cpp

```diff
@@ -5,19 +5,91 @@
 
 #include <dune/common/parallel/mpihelper.hh>
 #include <dune/common/test/testsuite.hh>
+
 using Dune::TestSuite;
 
 #include "testFEElement.hh"
 
 #include <dune/functions/functionspacebases/basistags.hh>
 #include <dune/functions/functionspacebases/lagrangebasis.hh>
+#include <dune/functions/functionspacebases/lagrangedgbasis.hh>
 #include <dune/functions/functionspacebases/powerbasis.hh>
 
-#include "ikarus/finiteElements/mechanics/linearElastic.hh"
+#include <ikarus/finiteElements/mechanics/linearElastic.hh>
+#include <ikarus/utils/duneUtilities.hh>
+#include <ikarus/utils/eigenDuneTransformations.hh>
 
 template <typename Basis>
 using LinearElasticElement = Ikarus::LinearElastic<Basis>;
 
+auto checkCauchyStress2D() {
```

Comment:
This functionality should be in src/easTest.hh for Displacementbased and for the already defined " Ikarus::EnhancedAssumedStrains<DisplacementBasedElement>" then you need a lot les of boilerplate code.
But the element corners are random. Thus you can not use such an easy `expectedStress`vector.

We can talk about this tomoorow